concent_api/utils/decorators.py

```diff
@@ -276,3 +278,18 @@ def wrapper(request, golem_message, client_public_key):
         response_from_view = view(request,  golem_message, client_public_key)
         return response_from_view
     return wrapper
+
+
+def log_task_errors(task):
+
+    @wraps(task)
+    def wrapper(*args, **kwargs):
+        try:
+            return task(*args, **kwargs)
+        except Exception as exception:
+            if settings.DEBUG is True:
+                crash_logger.error(
+                    f'Exception occurred while executing task {task.__name__}: {exception}, Traceback: {traceback.print_exc()}'
+                )
```

Comment:
No!!! ;) We definitely don't want to disable error reports on the clusters (all our clusters are running with `DEBUG = False`). If anything it would be the opposite - don't send mails when running locally but send them when running on the cluster.

But this should not depend on this setting at all. It's up to the handler to suppress the report if you're running in debug mode.